terraform/main.tf

```diff
@@ -79,16 +79,17 @@ module "postgres_db_development" {
   source = "github.com/LBHackney-IT/aws-hackney-common-terraform.git//modules/database/postgres"
   environment_name = "development"
   vpc_id = data.aws_vpc.development_vpc.id
-  db_name = "mosaic-mirror"
-  db_port  = 5002
-  subnet_ids = data.aws_subnet_ids.development_private_subnets.ids
   db_engine = "postgres"
   db_engine_version = "12.2"
+  db_identifier = "mosaic-mirror"
   db_instance_class = "db.t2.micro"
+  db_name = "mosaic-mirror"
```

Comment:
`db_identifier` variable still now required, as per: https://github.com/LBHackney-IT/aws-hackney-common-terraform/commit/b9c6300e3734cea3a102f136ecdc43c9721c9177#diff-747a367b56b74b6e938e5119f00436c5R19

Checking now whether `db_name` still needed here.

Reply:
(Looks like `db_name` still [very important](https://github.com/LBHackney-IT/aws-hackney-common-terraform/blob/b9c6300e3734cea3a102f136ecdc43c9721c9177/modules/database/postgres/main.tf#L32) so leaving in place here.)